external-crates/move/documentation/book/src/enums.md

```diff
@@ -0,0 +1,335 @@
+# Enumerations
+
+An _enum_ is a user-defined data structure containing one-or-more _variants_. Each variant can
```

Comment:
```suggestion
An _enum_ is a user-defined data structure containing one or more _variants_. Each variant can
```
Hyphens not needed unless "one-or-more" describes the type of variants (adjective)?